src/components/wish/WishDetail.jsx

```diff
@@ -0,0 +1,108 @@
+import styled from 'styled-components';
+import HeartButton from 'components/common/HeartButton';
+import { client } from 'libs/api';
+
+function WishDetail({ detail, setWishDetail }) {
+  const toggleHeart = async (id, currentHeartStatus) => {
+    await client.patch(`/wish/${id}`, {
+      like: !currentHeartStatus,
+    });
+
+    const newDetail = detail.map((post) => {
+      if (post.id === id) {
+        return {
+          ...post,
+          like: !currentHeartStatus,
+        };
+      }
+
+      return post;
+    });
+
+    setWishDetail(newDetail);
+  };
+
+  return (
+    <StyledWishDetail>
+      {detail.map(({ id, image, title, price, like }) => (
+        <li key={id}>
+          <StyledDetailCard>
+            <div>
+              <img src={image} />
+              <HeartButton isLiked={like} toggleHeart={() => toggleHeart(id, like)} />
+            </div>
+            <span>{title}</span>
+            <div>
+              <span>₩{price.toLocaleString()}</span>
+              <span> / </span>
+              <span> 박</span>
```

Comment:
```suggestion
              <span>/</span>
              <span>박</span>
```
아래 보니까 어차피 span에 padding-left 들어가 있어서 띄어쓰기 없애도 돼!